nkms/crypto/keyring/keys/signing_keypair.py

```diff
@@ -0,0 +1,58 @@
+import msgpack
+from random import SystemRandom
+from py_ecc.secp256k1 import N, privtopub, ecdsa_raw_sign, ecdsa_raw_recover
+
+
+class SigningKeypair(object):
+    def __init__(self, privkey_bytes=None):
+        self.secure_rand = SystemRandom()
```

Comment:
Oh wow, didn't know that random provides this also :-)

Reply:
Yeah, `SystemRandom` is great. It saves a lot of headache from trying to implement similar functions with raw output from `os.urandom`.